### PR TITLE
I would like to add my city Minsk to map

### LINKED ISSUE #302: Minsk, Belarus to the map
## Let's add Minsk, Belarus to the map

Hi everyone! This issue is to track the addition of **MINSK** to the map, details are as follows:

- Longitude: 27.561524
- Latitude: 53.904540

Any Minskers are welcome to comment!

### PR DESCRIPTION
**Pull request by tyzia**
There is no Minsk on this map, let's fix it! closes #302

### CODE REVIEW

File: tyzia-Minsk.topojson [hunk 1]
```diff
@@ -0,0 +1 @@
+{ "type": "Feature", "geometry": {"type": "Point", "coordinates": 27.561524, 53.904540]}, "properties": {"username": "tyzia"} }
```
Comment (tyzia) [inline tyzia-Minsk.topojson]: Why it is red?

**Comment by tyzia** [comment]
Can't merge my pull request, the button is not active (gray). Any ideas how to fix it?

**Comment by tyzia** [comment]
github

**Comment by brianamarie** [comment]
@beardofedu @hollenberry @crichID @hectorsector Any ideas why @tyzia isn't allowed to merge?

**Comment by tyzia** [comment]
Just double checked and found that I can merge pull request (have green button) with other pull request (@githubteache  #277 )

**Comment by brianamarie** [comment]
@tyzia 🎉 That's great news!! It looks good to me, so feel free to merge! 🚢

**Comment by crichID** [comment]
@tyzia #277 is actually a reverse merge and not going to merge your PR into master.

**Comment by crichID** [comment]
@tyzia can you please do a quick check for me and confirm the following:

1. You are currently logged in to GitHub under your `tyzia` account.
2. You have verified email addresses (check this by going to settings > Emails)
3. You do not need to be whitelisted? (check this by looking for a banner on the top of your screen)

You should be able to merge this on your own if all of the above are OK!

**Comment by tyzia** [comment]
@crichID

1. Yes
2. Yes
3. No banners on top
Still not able to merge ))

**Comment by tyzia** [comment]
Please check
github2

**Comment by tyzia** [comment]
Button 'Merge pull request' still not active.

**Comment by brianamarie** [comment]
Ok @tyzia, we will look into it further. Thank you for your patience! 😄

**Comment by tyzia** [comment]
@brianamarie thank you. It should be a nice start for me using github by solving this problem finaly.

**Comment by crichID** [comment]
@tyzia can you please check again? We were playing with some setting this morning and may have accidentally left the wrong switch flipped!

**Comment by tyzia** [comment]
Super @crichID I re-logged in and the button is green! Thank you!

**Comment by tyzia** [comment]
Shall I delete this branch? Will it delete all this comments?

**Comment by brianamarie** [comment]
@tyzia Yep, go ahead and delete the branch. Don't worry - the comments and all of the history will still be here.

**Comment by tyzia** [comment]
@brianamarie thanks, appreciate your help!

**Pull request merged**